codesystems/CodeSystem-England-MessageEventBARS.xml

```diff
@@ -0,0 +1,47 @@
+<CodeSystem xmlns="http://hl7.org/fhir">
+    <id value="England-MessageEventBARS" />
+    <url value="https://fhir.nhs.uk/CodeSystem/England-MessageEventBARS" />
+    <identifier>
```

Comment:
should this be included? We don't include them in UK Core

Reply:
This is common in NHSE. I will document it and raise at second level review.